pkg/controller/volume/attachdetach/openshift_patch.go

```diff
@@ -0,0 +1,18 @@
+package attachdetach
+
+import (
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/component-base/featuregate"
+	"k8s.io/kubernetes/pkg/features"
+)
+
+func patchADCFeatureGates() featuregate.FeatureGate {
```

Comment:
patchADCFeatureGates suggests that this is has side-effect. Instead call it `patchedADCFeatureGates` or `customizedADCFatureGates`.